public/app/components/fontViewer/fontViewer.js

```diff
@@ -36,11 +37,20 @@ angular.module('kibibitCodeEditor')
         } else {
           vm.font = font;
           onFontLoaded();
+          getQuotes();
         }
 
       });
     };
 
+    function getQuotes() {
```

Comment:
this function is redundent. `QuotesService.getQuotes()` already logs the error if an error is returned.

So I think applying this inline might be easier to understand

```javascript
} else {
  vm.font = font;
  onFontLoaded();
  QuotesService.getQuotes().then(function(quoteList) {
    vm.previewText = quoteList[0];
  });
}
```